src/ctia/stores/es/crud.clj

```diff
@@ -535,21 +535,21 @@ It returns the documents with full hits meta data including the real index in wh
   [{:keys [aggregate-on granularity timezone]
     :or {timezone "+00:00"}} :- HistogramQuery]
   {:date_histogram
-   {:field aggregate-on
+   {:field (get enumerable-fields-mapping aggregate-on aggregate-on)
```

Comment:
It should only be applied to topn and cardinality.
Topn and cardinality performs aggregate on keywords field while histograms aggregate on time field which are always indexed as date.
```suggestion
   {:field aggregate-on)
```